sympy/integrals/rubi/tests/test_utility_function.py

```diff
@@ -1495,7 +1495,6 @@ def test_TrigToExp():
     assert TrigToExp(sin(x)) == -I*(exp(I*x) - exp(-I*x))/2
     assert TrigToExp(cos(x)) == exp(I*x)/2 + exp(-I*x)/2
     assert TrigToExp(cos(x)*tan(x**2)) == I*(exp(I*x)/2 + exp(-I*x)/2)*(-exp(I*x**2) + exp(-I*x**2))/(exp(I*x**2) + exp(-I*x**2))
-    assert TrigToExp(cos(x)*tan(x**2)*sin(x)**2) == -I*(exp(I*x)/2 + exp(-I*x)/2)*(exp(I*x) - exp(-I*x))**2*(-exp(I*x**2) + exp(-I*x**2))/(4*(exp(I*x**2) + exp(-I*x**2)))
```

Comment:
Okay, then simpify the integers in the answer and do `simplify(TrigToExp(exp) - answer)) == 0`